packages/frontend/tailwind.config.js

```diff
@@ -1,5 +1,8 @@
 /** @type import("tailwindcss/types").Config */
 module.exports = {
+  future: {
+    hoverOnlyWhenSupported: true,
```

Comment:
please add a comment with explaination, I assume it was added to prevent hover events from happening on mobile?

Reply:
yup, basically its going to be default in tailwind v4